### ISSUE TITLE
Changing computer make the pretrained model fail

### ISSUE THREAD

**Issue by AnlanSun** [tensorflow/tensorflow#7908]
I switched to another computer to keep training my model .But I found the loss is as high as random init after loading the pretrained model. I trained overagain on this new computer and the restore seems work.What's wrong? Hope for help.

using TF:1.0

**Comment by aselle** [comment]
Please provide details about what platform you are using  (operating system, architecture). Also include your TensorFlow version. Also, include what model you are trying to train and how your invoking it, moving it to another machine, etc.. We ask for this in the issue submission template, because  it is really difficult to help without that information. Thanks!

**Comment by AnlanSun** [comment]
Thanks for response! I am using the latest version of TF(just installed  4 days ago). On Ubuntu 14.04. Using vgg16 and a InceptionV1 with just 4 conv layers. Both of them show this problem.
The part of Saving and reloading is shown bellow
`    init_op = tf.initialize_all_variables()

    saver = tf.train.Saver()
    with tf.Session() as sess:
        #sess.run(init_op)
        #new_saver = tf.train.import_meta_graph('/mnt/data2/sunanlan/sunanlan/DGD/check25_rota_guiyi/lr_0.031_step_39.ckpt.meta')

        #print( tf.train.latest_checkpoint('./'))

        #what=new_saver.restore(sess,'/mnt/data2/sunanlan/sunanlan/DGD/check25_rota_guiyi/lr_0.031_step_39.ckpt')
        saver.restore(sess, "/mnt/data2/sunanlan/sunanlan/DGD/from_2/step_66.ckpt")
        tf.train.start_queue_runners(sess=sess)
        for epoch in range(100):
            widgets = ["train D epoch #%d|" % epoch, Percentage(), Bar(), ETA()]
            pbar = ProgressBar(maxval=1000, widgets=widgets)
            pbar.start()
            loss_avg = 0

            for step in range(1001):
                pbar.update(step)
                imgs_n,labels_n = readdatas.read(lengbat,160,70,image_paths,labels_in,classes)
                global is_train
                is_train = True


                _, loss_value,lrt,img_all = sess.run([train_tensor,total_loss,lr,img_new],{imgs:imgs_n,labels:labels_n})
                loss_avg += loss_value
                if step % 50 == 0:
                    avg = loss_avg / 50
                    print("loss: %f " % (avg))
                    print("lr: %f" %(lrt))
                    loss_avg = 0
                if step == 1000:
                     saver_path = saver.save(sess, "./from_2/step_"+str(epoch)+".ckpt")
`

**Comment by AnlanSun** [comment]
I just use `scp -r ` to download it on another conputer. Is that wrong? And what shocked me today is I trained , reloaded InceptionV1 with 4 conv layers on the same computer and got the same phenomenon

**Comment by prb12** [comment]
@AnlanSun Your code seems to write checkpoints based on epoch number.  You load epoch 66, then start training from epoch=0 and overwrite checkpoints every 1000 steps.  Are you sure you didn't overwrite your own checkpoint file at some point?

**Comment by prb12** [comment]
@AnlanSun Your code appears to generate checkpoint filenames numbered by `epoch`.  You load the checkpoint from epoch66 and then start counting from zero, saving a new checkpoint every 1000 training steps.  Are you sure you didn't accidentally overwrite your checkpoint file at some point?

PS. Copying the file, e.g. using `scp` ought to be just fine.

**Comment by AnlanSun** [comment]
@prb12 Thanks for the response. I think I did not make that mistake. Because even it overwrite the existing file the loss should not start from that high. But is it a problem to overwrite a file?  Or while I am training a model loaded form like epoch66 , Is it fine to overwrite that expoch66 file?

**Comment by AnlanSun** [comment]
@prb12 I would like to send you all my code. Cause I really can not find the mistake about that. Is ti possible to get you email? Hope for help

**Comment by aselle** [comment]
You can post your code, @AnlanSun, but you will have the largest chance of someone finding your problem if you reduce the size of your code to the smallest piece of code that can reproduce the problem. That exercise of reduction also stands a great chance of allowing you to re-check your code for errors. Please do that first and post your code.

**Comment by aselle** [comment]
Also, This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!

**Comment by girving** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
